src/components/Card/CardWithoutPicture.js

```diff
@@ -1,88 +1,96 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
+import CardActions from "@material-ui/core/CardActions";
+import Avatar from "@material-ui/core/Avatar";
+import IconButton from "@material-ui/core/IconButton";
+import Typography from "@material-ui/core/Typography";
+import { red } from "@material-ui/core/colors";
+import Chip from "@material-ui/core/Chip";
+import ShareOutlinedIcon from "@material-ui/icons/ShareOutlined";
+import ChatOutlinedIcon from "@material-ui/icons/ChatOutlined";
+import TurnedInNotOutlinedIcon from "@material-ui/icons/TurnedInNotOutlined";
+import MoreVertOutlinedIcon from "@material-ui/icons/MoreVertOutlined";
+import ToggleButton from "@material-ui/lab/ToggleButton";
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+import KeyboardArrowUpIcon from "@material-ui/icons/KeyboardArrowUp";
+import KeyboardArrowDownIcon from "@material-ui/icons/KeyboardArrowDown";
+import {
+  createTheme,
+  responsiveFontSizes,
+  ThemeProvider
+} from "@material-ui/core/styles";
 
-const useStyles = makeStyles((theme) => ({
+const useStyles = makeStyles(theme => ({
   root: {
-    width: '45%',
-    borderRadius: '10px',
-    [theme.breakpoints.down('md')]: {
-      width: '80%',
-    },
-    [theme.breakpoints.down('xs')]: {
-      width: '99%',
+    width: "45%",
+    borderRadius: "10px",
+    [theme.breakpoints.down("md")]: {
+      width: "80%"
     },
+    [theme.breakpoints.down("xs")]: {
+      width: "99%"
+    }
   },
   grow: {
-    flexGrow: 1,
+    flexGrow: 1
   },
   margin: {
-    marginRight: '5px',
+    marginRight: "5px"
   },
   expandOpen: {
-    transform: 'rotate(180deg)',
+    transform: "rotate(180deg)"
   },
   avatar: {
-    backgroundColor: red[500],
+    backgroundColor: red[500]
   },
   inline: {
-    fontWeight: 600,
+    fontWeight: 600
   },
   contentPadding: {
-    padding: '0 16px',
+    padding: "0 16px"
   },
   icon: {
-    padding: '5px',
+    padding: "5px"
   },
   time: {
-    lineHeight: '1',
+    lineHeight: "1"
   },
   small: {
-    padding: '4px',
-  },
+    padding: "4px"
+  }
 }));
 
 export default function CardWithoutPicture(props) {
   const classes = useStyles();
   let theme = createTheme();
   theme = responsiveFontSizes(theme);
-  const [alignment, setAlignment] = React.useState('left');
+  const [alignment, setAlignment] = React.useState("left");
   const [count, setCount] = useState(1);
   const handleIncrement = () => {
     setCount(count + 1);
-  }
+  };
 
   const handleDecrement = () => {
     setCount(count - 1);
-  }
+  };
 
   const handleAlignment = (event, newAlignment) => {
     setAlignment(newAlignment);
   };
 
   return (
-    <Card className={classes.root}>
+    <Card className={props.className ? props.className : classes.root}>
```

Comment:
you can re-write this using [JS Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).

`{props?.className || classes.root}`

Reply:
sure, done